pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -378,10 +377,11 @@ func (i *CommandInitOption) initKarmadaAPIServer() error {
 	if err := util.CreateOrUpdateService(i.KubeClientSet, i.karmadaAggregatedAPIServerService()); err != nil {
 		klog.Exitln(err)
 	}
+	karmadaAggregatedAPIServerDeployment := i.makeKarmadaAggregatedAPIServerDeployment()
 	if _, err := i.KubeClientSet.AppsV1().Deployments(i.Namespace).Create(context.TODO(), i.makeKarmadaAggregatedAPIServerDeployment(), metav1.CreateOptions{}); err != nil {
 		klog.Warning(err)
 	}
-	if err := WaitPodReady(i.KubeClientSet, i.Namespace, utils.MapToString(aggregatedAPIServerLabels), 30); err != nil {
+	if err := WaitForDeploymentRollout(i.KubeClientSet, karmadaAggregatedAPIServerDeployment, 30); err != nil {
```

Comment:
Some places use numbers directly, some places use `waitPodReadyTimeout`. Can we make them unified?